logger-service/logger-service.js

```diff
@@ -74,9 +74,56 @@ class Logger {
 	}
 }
 
+class dummyLogger {
+
+	constructor() {
+	}
+
+	log() {
+		return;
+	}
+
+	logToConsole(args) {
+		return;
+	}
+
+	logToFile(args) {
+		return;
+	}
+
+	storeOutputParameter(value) {
+		return;
+	}
+
+	getOutputParameterValue() {
+		return;
+	}
+
+	updateOutputCache(value) {
+		return;
+	}
+
+	clearOutputCache() {
+		return;
+	}
+
+	removeOutputStorage() {
+		return;
+	}
+}
+
 const logger = new Logger();
 
-module.exports = {
-	logger,
-	AppenderTypes
-};
\ No newline at end of file
+if (typeof window === 'undefined') {
```

Comment:
Refactor this no too many duplicated code. You can easily define `let logger` above and just have a logic over this variable nstead of the whole module.exports